app/routes/languages.rb

```diff
@@ -70,6 +70,11 @@ def topic_not_found(topic)
         status 404
         erb :"languages/not_found", locals: { track_id: topic }
       end
+
+      def active(topic, selected_topic)
+        'active' if topic == selected_topic
+      end
```

Comment:
I really dislike the idea of having method definitions in views. I'd rather have the conditional.

I'm going to change this back and merge it locally (I don't think it's fair to make @nickborromeo thrash more on this bit of code).